execution_engine/Cargo.toml

```
@@ -14,6 +14,7 @@ license = "Apache-2.0"
 anyhow = "1.0.33"
 base16 = "0.2.1"
 bincode = "1.3.1"
+blake3 = { version = "1.5.0", default-features = false }
```

I think it may actually be better to use default features i.e. 

```suggestion
blake3 = "1.5.0"
```

According to the source here: https://github.com/BLAKE3-team/BLAKE3/blob/92e4cd71be48fdf9a79e88ef37b8f415ec5ac210/Cargo.toml#L21-L26 if you use `--no-default-features` it will not do runtime CPU feature detection and won't use SIMD implementation.

---

Good catch! Suggestion approved: https://github.com/casper-network/casper-node/pull/4412/commits/8bd8fbb7893e74846c65468fe1518646eca62071.